config/api/src/main/java/org/jboss/windup/config/metadata/LabelMetadataBuilder.java

```diff
@@ -0,0 +1,42 @@
+package org.jboss.windup.config.metadata;
+
```

Comment:
Please add comments to describe the class and the methods as well.

Reply:
done